back/src/Guide/Types/Core.hs

```diff
@@ -179,23 +179,32 @@ data Item = Item {
   _itemName        :: Text,            -- ^ Item title
   _itemCreated     :: UTCTime,         -- ^ When the item was created
   _itemGroup_      :: Maybe Text,      -- ^ Item group (affects item's color)
-  _itemDescription :: MarkdownBlock,   -- ^ Item summary
+  _itemHackage     :: Maybe Text,      -- ^ Package name on Hackage
+  _itemSummary     :: MarkdownBlock,   -- ^ Item summary
   _itemPros        :: [Trait],         -- ^ Pros (positive traits)
   _itemProsDeleted :: [Trait],         -- ^ Deleted pros go here (so that
                                        --   it'd be easy to restore them)
   _itemCons        :: [Trait],         -- ^ Cons (negative traits)
   _itemConsDeleted :: [Trait],         -- ^ Deleted cons go here
   _itemEcosystem   :: MarkdownBlock,   -- ^ The ecosystem section
   _itemNotes       :: MarkdownTree,    -- ^ The notes section
-  _itemLink        :: Maybe Url,       -- ^ Link to homepage or something
-  _itemKind        :: ItemKind         -- ^ Is it a library, tool, etc
+  _itemLink        :: Maybe Url        -- ^ Link to homepage or something
   }
   deriving (Show, Generic, Data)
 
-deriveSafeCopySorted 11 'extension ''Item
+deriveSafeCopySorted 12 'extension ''Item
 makeFields ''Item
 
-changelog ''Item (Current 11, Past 10) []
+changelog ''Item (Current 12, Past 11)
+  [Removed "_itemKind"  [t|ItemKind|],
+   Added "_itemHackage" [hs|case _itemKind of
```

Comment:
This should also mention that description has been renamed to summary. I'll do it by myself.